client/src/pages/AdminLogin/styles.js

```diff
@@ -0,0 +1,24 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
I am ok using this type of styling. However, there needs to be a CSS standard. I would be super on board to change my css modules to styles.js

Reply:
Agreed.